.github/workflows/staging-build-windows.yml

```diff
@@ -280,7 +280,7 @@ jobs:
           echo "running tests"
           cd ..\alerting\alerting
           dir
-          ../gradlew.bat integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername=es-integrationtest -Dhttps=true -Duser=admin -Dpassword=admin
+          ..\gradlew.bat integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername=es-integrationtest -Dhttps=true -Duser=admin -Dpassword=admin
```

Comment:
Line 116 has spaces after the "-D"; this line does not. Looking at the failed test run, it seems like you need the spaces.

Reply:
Looks like your latest commit fixes this. Disregard.

Reply:
checking

Reply:
fixed and worked, please review.